docs/references/expo/offline-support.mdx

```diff
@@ -71,3 +71,49 @@ To enable offline support in your Expo app, follow these steps:
   }
   ```
 </Steps>
+
+## How to handle network errors
+
+To handle network errors in your Expo app, you can use the `isClerkRuntimeError()` function to check if the error is a Clerk-related error. Clerk-related errors are returned as an array of [`ClerkAPIError`](/docs/references/javascript/types/clerk-api-error) objects. These errors contain a `code` property that you can use to identify the specific error. See the following example.
+
+```tsx {{ mark: [1, [23, 31]] }}
+import { isClerkRuntimeError } from '@clerk/clerk-js'
```

Comment:
can we re-export this for the clerk-expo package? for example, for next.js, we export `isClerkAPIResponseError` from `clerk/nextjs/errors`

Reply:
Seconded! Please do this.

Reply:
I will do that!

Reply:
Opened a PR here: https://github.com/clerk/javascript/pull/4815

It needs to be merged and then released!

Reply:
Merged! Needs to be released in the next SDK release!

Reply:
The package is released! We can merge this PR when it's ready!